centml/compiler/config.py

```diff
@@ -24,5 +24,11 @@ class Config:
     BACKEND_BASE_PATH = os.path.join(CACHE_PATH, "backend")
     SERVER_BASE_PATH = os.path.join(CACHE_PATH, "server")
 
+    # Use a constant path since torch.save uses the given file name in it's zipfile.
+    # Thus, a different path would result in a different hash.
+    SERIALIZED_MODEL_FILE = "serialized_model.pkl"
+    SERIALIZED_INPUT_FILE = "serialized_input.pkl"
```

Comment:
if the serialized files are zips, can we name them with .zip ext?

Reply:
yup makes sense